src/model.py

```diff
@@ -383,15 +413,12 @@ async def generate(self, request):
                             len(prev_output.text)
                             for prev_output in prev_outputs.outputs
                         ]
+                    response = self.create_stream_response(output, prev_outputs_lengths)
+                    flags = 0
                     if output.finished:
-                        response_sender.send(
-                            self.create_stream_response(output, prev_outputs_lengths),
-                            flags=pb_utils.TRITONSERVER_RESPONSE_COMPLETE_FINAL,
-                        )
-                    else:
-                        response_sender.send(
-                            self.create_stream_response(output, prev_outputs_lengths)
-                        )
+                        flags = pb_utils.TRITONSERVER_RESPONSE_COMPLETE_FINAL
+                        decrement_ongoing_request_count = False
+                    self._response_queue.put_nowait((response_sender, response, flags))
```

Comment:
Can `_response_queue` be full here?

Reply:
I think the queue size is unlimited, by [initializing the queue from its default arguments](https://github.com/triton-inference-server/vllm_backend/pull/46/files#diff-6159e4e86b1b15ffd250a78dbdf29a41b004b4963154e49f8b96609b8ea977ceR119).
https://docs.python.org/3.10/library/queue.html#queue.Queue